controller.go

```diff
@@ -280,13 +282,21 @@ func loadSettings() error {
 		Default("text/plain").StringVar(&denyInternalRespContentType)
 	kingpin.Flag("deny-internal-domains-response-status-code", "Defines the response status code for a request identified as to an internal domain when -deny-internal-domains is set.").
 		Default("401").IntVar(&denyInternalRespStatusCode)
+	kingpin.Flag("target-access-mode", "Target group accessing Ingress via HostPort or AWS VPC CNI. Set to ASG for HostPort access or CNI for pod direct IP access.").
+		Default(aws.TargetAccessModeHostPort).EnumVar(&targetAccessMode, aws.TargetAccessModeHostPort, aws.TargetAccessModeAWSCNI)
+	kingpin.Flag("target-cni-pod-selector", "AWS VPC CNI only. Defines the query (namespace/labelselector) for ingress pods that should be linked to target group. Format 'namespace/key=value'.").
+		Default(aws.DefaultTargetCNILabelSelector).StringVar(&targetCNIPodSelector)
```

Comment:
Maybe we should split this into two flags for namespace and for label for clarity.
I would also remove `target-` prefix and only use `cni-` prefix similar to `alb-` and `nlb-`.

Then we may also allow repeatable label flag (maybe as a next improvement) like
```
--cni-namespace=kube-system
--cni-label=application=skipper-ingress
--cni-label=component=ingress
```
E.g. we use the same `application=skipper-ingress` label for multiple components so without multi-label support we would need to introduce a special label only for the CNI selector.

Reply:
I would also remove `DefaultTargetCNILabelSelector` and check that of CNI is enabled that namespace and at least one label are configured.
Also `DefaultTargetCNILabelSelector` uses `skipper-ingress` but controller is mostly agnostic to the actual ingress implementation (`skipper` string is not used anywhere but in docs and tests)

Reply:
CNI (common network interface) is a different term (tools that implement are for example flannel, aws-vpc-cni, calico, ..) and I would not understand the flag name if changed.
I think the flag split is good, but then use:
```
--target-cni-namespace=kube-system
--target-cni-pod-label=application=skipper-ingress
--target-cni-pod-label=component=ingress
```

Reply:
Actions:
* removing the default
* splitting up the flags
* supporting list of labels

Question: technically `labelselector` is the term, should we use that instead of `label` (which sounds more like an assignment to me)